deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -128,13 +128,21 @@ impl MessageObject {
         let file_bytes = message.get_filebytes(context).await?.unwrap_or_default();
         let override_sender_name = message.get_override_sender_name();
 
+        let parent_message = message.parent(context).await?;
+        let parent_msg_id = parent_message.as_ref().map(|m| m.get_id());
+
         let webxdc_info = if message.get_viewtype() == Viewtype::Webxdc {
             Some(WebxdcMessageInfo::get_for_message(context, msg_id).await?)
+        } else if message.get_info_type() == deltachat::mimeparser::SystemMessage::WebxdcInfoMessage
```

Comment:
```Rust
        } else if let (deltachat::mimeparser::SystemMessage::WebxdcInfoMessage, Some(parent_msg)) = (message.get_info_type(), message.parent(context).await?) {
```
This way the parent message is only loaded when needed and `unwrap()` is avoided